renderer.php

```diff
@@ -162,7 +162,9 @@ public function active_factors() {
                 $timecreated  = $userfactor->timecreated == '-' ? '-'
                     : userdate($userfactor->timecreated,  get_string('strftimedatetime'));
                 $lastverified = $userfactor->lastverified;
-                if ($lastverified != '-') {
+                if ($lastverified == 0) {
+                    $lastverified = '-';
```

Comment:
would this be better as a string saying 'Not yet verified' ?

Reply:
Split this to a new issue here:
https://github.com/catalyst/moodle-tool_mfa/issues/346